plugins/rsyslog.yaml

```diff
@@ -49,17 +79,31 @@ parameters:
     hidden: true
 
 # Set Defaults
+# {{$source := default "udp" .source}}
```

Comment:
I think we should make `source` a required value so users are prompted to select file / udp. It might be easy for them to miss the file option if udp is set by default.